src/utils/typography.js

```diff
@@ -1,6 +1,8 @@
 import Typography from "typography"
 
 const typography = new Typography({
+  baseFontSize: "14px",
+  baseLineHeight: "20px",
   headerFontFamily: ["Nunito", "SF Pro Rounded"],
   bodyFontFamily: ["Nunito", "SF Pro Rounded"],
```

Comment:
Change to `["Nunito", "SF Pro Rounded", "sans-serif"]` because we need to have an OS-native fallback.

Reply:
Done.